src/build.py

```diff
@@ -1256,10 +1276,10 @@ def TestEmtestAsm2Wasm():
 
 ALL_TESTS = [
     Test('bare', TestBare),
-    Test('asm', TestAsm),
-    Test('emwasm', TestEmwasm),
-    Test('emtest', TestEmtest),
-    Test('emtest-asm', TestEmtestAsm2Wasm),
+    Test('asm', TestAsm, no_windows=True),
```

Comment:
I legitimately like how the default for these is `False`, even though that means more line changes here, because of the consistency with `Source` and `Build`